### PR TITLE
[HDFS] Update documentation

### PR DESCRIPTION
**Pull request by gabrielhartmann**

### CODE REVIEW

File: frameworks/hdfs/docs/api-reference.md [hunk 1]
```diff
@@ -29,117 +29,283 @@ If you are using Enterprise DC/OS, the security mode of your installation may al
...
+$ curl -X POST -H "Authorization:token=<auth_token>" <dcos_url>/service/hdfs/v1/plans/deploy/interrupt
 ```
 
 ## Resume Installation
 
 The REST API request below will resume installation at the next pending node.
 
 ```bash
-$ curl -X PUT <dcos_surl>/service/hdfs/v1/plan/continue
+$ curl -X PUT <dcos_surl>/service/hdfs/v1/plans/deploy/continue
```
Comment (triclambert) [inline frameworks/hdfs/docs/api-reference.md]: s/surl/url/

File: frameworks/hdfs/docs/api-reference.md [hunk 2]
```diff
@@ -29,117 +29,283 @@ If you are using Enterprise DC/OS, the security mode of your installation may al
...
+The contents of the responses represent valid hdfs-site.xml and core-site.xml that can be used by clients to connect to the service.
 
 # Nodes API
 
-The nodes API provides endpoints for retrieving information about nodes, restarting them, and replacing them.
+The pods API provides endpoints for retrieving information about nodes, restarting them, and replacing them.
 
 ## List Nodes
 
 A list of available node ids can be retrieved by sending a GET request to `/v1/state/tasks`:
```
Comment (triclambert) [inline frameworks/hdfs/docs/api-reference.md]: s/state\/tasks/pods/

File: frameworks/hdfs/docs/api-reference.md [hunk 3]
```diff
@@ -148,276 +314,408 @@ $ curl  -H "Authorization:token=<auth_token>" <dcos_url>/service/hdfs/v1/state/t
 You can retrieve node information by sending a GET request to `/v1/state/tasks/info/<node-id>`:
```
Comment (triclambert) [inline frameworks/hdfs/docs/api-reference.md]: s/state\/tasks/pods/

File: frameworks/hdfs/docs/api-reference.md (hunk 1 above)
Comment (gabrielhartmann) [inline frameworks/hdfs/docs/api-reference.md]: I don't understand.  This URL is accurate.

**Pull request merged**